ethcore/src/state/substate.rs

```diff
@@ -45,10 +50,20 @@ impl Substate {
 	/// Merge secondary substate `s` into self, accruing each element correspondingly.
 	pub fn accrue(&mut self, s: Substate) {
 		self.suicides.extend(s.suicides.into_iter());
+		self.garbage.extend(s.garbage.into_iter());
```

Comment:
is it guaranteed that the union of two substates won't contain the same address in two sets?